apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -194,6 +194,8 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         state?.organization?.organizationType);
     const currentOrganization: OrganizationResponseInterface = useSelector((state: AppState) =>
         state.organization.organization);
+    const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state?.config?.ui?.features);
```

Comment:
Lets straightaway access `disabledFeatures`.
```suggestion
    const disabledFeatures: FeatureConfigInterface = useSelector((state: AppState) => state?.config?.ui?.features?.applications?.disabledFeatures);
```

Reply:
Fixed with c09d1f9